src/mozanalysis/experiment.py

```diff
@@ -563,6 +626,74 @@ def _build_enrollments_query_glean_event(self, time_limits, dataset):
             dataset=self.app_id or dataset,
         )
 
+    def _build_exposure_query_normandy(self, time_limits):
+        """Return SQL to query exposures for a normandy experiment"""
+        return """
+        SELECT
+            e.client_id,
+            e.branch,
+            min(e.submission_date) AS exposure_date,
+            COUNT(e.submission_date) AS num_exposure_events
+        FROM raw_enrollments re
+        LEFT JOIN (
+            SELECT
+                client_id,
+                `mozfun.map.get_key`(event_map_values, 'branch') AS branch,
+                submission_date
+            FROM
+                `moz-fx-data-shared-prod.telemetry.events`
+            WHERE
+                event_category = 'normandy'
+                AND (event_method = 'exposure' AND event_method = 'expose')
+                AND submission_date
+                    BETWEEN '{first_enrollment_date}' AND '{last_enrollment_date}'
+                AND event_string_value = '{experiment_slug}'
+        ) e
+        ON re.client_id = e.client_id AND
+            re.branch = e.branch AND
+            e.submission_date >= re.enrollment_date
```

Comment:
This wasn't explicitly mentioned in the proposal, but do we only consider the exposures after the enrollment or should exposures during the enrollment window (which I guess could happen before an enrollment event was received) be considered?

Reply:
After enrollment sounds right to me. If the exposure event isn't experiment-specific, then we don't want to include exposure that didn't reflect the experiment content.